### PR TITLE
Feature/master/upgrade angular

### PR DESCRIPTION
**Pull request by Waseem826**
This PR upgrades Angular to version 8

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,20 +29,18 @@
...
 /.sass-cache
 /connect.lock
 /coverage
 /libpeerconnection.log
 npm-debug.log
+yarn-error.log
 testem.log
 /typings
 
-# e2e
```
Comment (travisspencer) [inline .gitignore]: This could be added back

File: browserslist [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```
Comment (travisspencer) [inline browserslist]: is this needed for the sample to run?

File: e2e/protractor.conf.js [hunk 3]
```diff
@@ -0,0 +1,32 @@
+// @ts-check
```
Comment (travisspencer) [inline e2e/protractor.conf.js]: the sample doesn't have any end-to-end tests, does it? If not, this can be removed.

File: karma.conf.js [hunk 4]
```diff
@@ -0,0 +1,32 @@
+// Karma configuration file, see link for more information
```
Comment (travisspencer) [inline karma.conf.js]: the sample doesn't have any unit tests, does it? If not, this can be removed.

File: browserslist (hunk 2 above)
Comment (Waseem826) [reply browserslist]: No, but it was added by Angular CLI so I kept it.

File: .gitignore (hunk 1 above)
Comment (Waseem826) [reply .gitignore]: I have removed the `e2e` folder so that's not needed

**Pull request merged**